README.md

```diff
@@ -4,3 +4,27 @@ fwbuilder
 =========
 
 Firewall Builder is a GUI firewall management application for iptables, PF, Cisco ASA/PIX/FWSM, Cisco router ACL and more. Firewall configuration data is stored in a central file that can scale to hundreds of firewalls managed from a single UI.
+
+
+Installation instructions
+=========================
+
+
+Ubuntu
+---------
+
+run the following apt-get install command to install the compile requirements:
+
+ sudo apt-get install git automake autoconf libtool libxml2-dev libxslt-dev libsnmp-dev qt4-dev-tools
+ git clone https://github.com/fwbuilder/fwbuilder.git
+ cd fwbuilder
+ ./autogen.sh
```

Comment:
If you're compiling a custom version, you may add an installation prefix, e.g. `--prefix=/usr/local` or `--prefix=/opt` to not collide with binaries from distribution packages.